src/main/java/no/nav/pto/veilarbportefolje/opensearch/OpensearchQueryBuilder.java

```diff
@@ -465,6 +469,8 @@ static SearchSourceBuilder sorterQueryParametere(String sortOrder, String
             case "barn_under_18_aar" ->
                     sorterBarnUnder18(searchSourceBuilder, order, brukerinnsynTilganger, filtervalg);
             case "brukersSituasjonSistEndret" -> searchSourceBuilder.sort("brukers_situasjon_sist_endret", order);
+            case "utdanningOgSituasjonSistEndret" ->
```

Comment:
this field `utdanning_og_situasjon_sist_endret` should be defined in opensearch_settings as `date`, otherwise sorting will not work